pylint/pyreverse/inspector.py

```diff
@@ -178,7 +179,17 @@ def visit_classdef(self, node: nodes.ClassDef) -> None:
             for assignattr in assignattrs:
                 if not isinstance(assignattr, nodes.Unknown):
                     self.handle_assignattr_type(assignattr, node)
-        # resolve implemented interface
```

Comment:
I think this comment should not be deleted, it just has to move down above line 193.